src/data/resolvers/metacolony.ts

```diff
@@ -0,0 +1,62 @@
+import { Resolvers } from '@apollo/client';
+
+import { Context } from '~context/index';
+import {
+  SubgraphColonyQuery,
+  SubgraphColonyQueryVariables,
+  SubgraphColonyDocument,
+} from '~data/index';
+
+import { getProcessedColony } from './colony';
+
+export const metaColonyResolvers = ({
+  colonyManager: { networkClient },
+  apolloClient,
+  ipfsWithFallback,
+}: Required<Context>): Resolvers => ({
+  Query: {
+    async processedMetaColony() {
+      try {
+        const metaColonyAddress = await networkClient.getMetaColony();
+        const metaColonyENSName = await networkClient.lookupRegisteredENSDomain(
+          metaColonyAddress,
+        );
+        if (metaColonyENSName) {
+          const { data } = await apolloClient.query<
+            SubgraphColonyQuery,
+            SubgraphColonyQueryVariables
+          >({
+            query: SubgraphColonyDocument,
+            variables: {
+              /*
+               * First convert it a string since in cases where the network name
+               * cannot be found via ENS it will throw an error
+               *
+               * Converting this to string basically converts the error object into
+               * a string form
+               */
+              address: metaColonyAddress.toLowerCase(),
```

Comment:
I spent too long trying to remember what I was trying to say here. Turns out I just copy-pasted this whole section and were it doesn't make sense :facepalm: 

It was from here: https://github.com/JoinColony/colonyDapp/blob/f53547841df4800f4a2d93d1b001027779a4efd7/src/data/resolvers/colony.ts#L314-L340

I'll remove the comment since it's confusing and wrong.